stripe/http_client.py

```diff
@@ -191,6 +215,29 @@ def _handle_request_error(self, e, url):
 class PycurlClient(HTTPClient):
     name = 'pycurl'
 
+    def __init__(self, verify_ssl_certs=True, proxy=None):
+        super(PycurlClient, self).__init__(
+            verify_ssl_certs=verify_ssl_certs, proxy=proxy)
+        # need to urlparse the proxy, since PyCurl
+        # consumes the proxy url in small pieces
+        if self._proxy:
+            # now that we have the parser, get the proxy url pieces
+            proxy = self._proxy
+            for scheme in proxy:
+                proxy[scheme] = urlparse(proxy[scheme])
+
+    def __get__proxy__(self, url):
```

Comment:
Can we move this to be prefixed by only a single underscore and then put it down with the other private methods below the public interface? (i.e. by `_handle_request_error`)? Thanks!